azkaban-common/src/main/java/azkaban/metric/AbstractMetric.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2012 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.metric;
+
+import org.apache.log4j.Logger;
+
+/**
+ * Abstract class for Metric
+ * @param <T> Type of Value of a given metric
+ */
+public abstract class AbstractMetric<T> implements IMetric<T>, Cloneable{
+  protected static final Logger _logger = Logger.getLogger(MetricReportManager.class);
```

Comment:
@erwa  suggested and It seemed a better convention so I used it. I have again changed back.